src/main/java/seedu/address/model/participant/Birthday.java

```diff
@@ -39,7 +39,12 @@ public static boolean isValidBirthday(String test) {
         if (month > 12 || month < 1) { //invalid month
             return false;
         }
-        if (year > Year.now().getValue()) { //invalid year
+        /**
+         * Valid DOB is based on the following:
+         * - Nobody is born in the future
+         * - No undergraduate is older than 50 years old
```

Comment:
Is this necessary?

Reply:
Yes. This is to prevent "absurd" data from being entered into the system.